cpp/src/parquet/CMakeLists.txt

```diff
@@ -408,6 +409,8 @@ add_parquet_test(arrow-reader-writer-test
                  arrow/arrow_statistics_test.cc
                  arrow/variant_test.cc)
 
+add_parquet_test(arrow-parquet-index-test SOURCES arrow/arrow_parquet_index_test.cc)
```

Comment:
Can we just call the source file `index_test.cc`? It's annoying to have these `arrow` and `parquet` prefixes.